notebooks/00_quick_start/lightgbm_tinycriteo.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>better to use a temporary directory in case data_path does not exist</p><p><br></p><p>example using context manager:</p><p>```</p><p>from tempfile import TemporaryDirectory</p><p><strong>with</strong> TemporaryDirectory() <strong>as</strong> tmp:</p><p>    save_file = os.path.join(tmp, 'finished.model')</p><p>    lgb_model.save_model(save_file)</p><p>    ....</p><p>```</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Fixed, thanks.